.circleci/config.yml

```diff
@@ -0,0 +1,19 @@
+version: 2
+jobs:
+  build:
+
+    # Specify the Xcode version to use
+    macos:
+      xcode: "9.3.0"
+
+    steps:
+      - checkout
+      - run:
+          name: Reset simulators with Snapshot
+          command: fastlane snapshot reset_simulators --force
```

Comment:
This bit should no longer be necessary, the issue was addressed at the Circle end a couple weeks ago